pkg/delete/delete.go

```diff
@@ -186,6 +204,22 @@ func deleteAutomations(c automationClient, automationResource config.AutomationR
 	return errors
 }
 
+func deleteBuckets(ctx context.Context, c bucketClient, entries []DeletePointer) []error {
```

Comment:
please add an AllBuckets variant and include it in the purge cmd as well - like the other AllX variants, list all objects and delete them.

Reply:
will do